src/commands/start.py

```diff
@@ -160,6 +166,12 @@
     prompt='Enter comma separated list of API endpoints for consensus nodes',
     help='Comma separated list of API endpoints for consensus nodes.',
 )
+@click.option(
+    '--graph-endpoint',
```

Comment:
Here and in other places, should it have a default value?

Reply:
set in https://github.com/stakewise/v3-operator/blob/bba355e56d487e7dd9e6572e08f402a62838cbe3/src/config/settings.py#L156. We don't know network here